#!/usr/bin/env python
PACKAGE = "kmm_gamepad"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("max_linear_vel", double_t, 0, "Maximum linear velocity, unit m/s", 0.2, 0, 1)
gen.add("max_angular_vel", double_t, 0, "Maximum angular velocity, unit rad/s", 0.5, 0, 2)
gen.add("deadzone_threshold", double_t, 0, "Gamepad deadzone threshold", 0.2, 0, 0.5)

exit(gen.generate(PACKAGE, "kmm_gamepad", "Gamepad"))
